GoogleDrive.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ * Description of GoogleDrive
+ *
+ * @author maxnivel
+ */
+class GoogleDrive {
+    public function __construct() {
```

Comment:
Não podemos encapsular as credenciais e forçar que esteja em um arquivo no mesmo diretório que a classe GoogleDrive.
1- Deve ser possível informar o path do arquivo de credencial.
2- Deve ser possível informar o access_token e refresh_token por parâmetro.

Reply:
-> O ideal é que tenha  a função loadCredentialsByFile($filePath) e loadCredentialByTokens($accessToken,$refreshToken).
-> Não usar constantes para armazenar as informações. Armazenar em atributos da própria classe.